images/linux/scripts/installers/pollinate.sh

```diff
@@ -4,12 +4,9 @@
 ##  Desc:  Installs Pollinate
 ################################################################################
 
+source $HELPER_SCRIPTS/invoke-tests.sh
+
 # Install Pollinate
 apt-get install -y --no-install-recommends pollinate
```

Comment:
Maybe move it to toolsets? So it will be installed along with other apt packages in [basic.sh](https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/installers/basic.sh)
And the same for `haveged`

Reply:
In that case, we have a few more candidates for such migration:
1. https://github.com/actions/virtual-environments/blob/7403f3318073a9a0d8ec04eca0cc11e16a3e8196/images/linux/scripts/installers/build-essential.sh#L9-L15
2. https://github.com/actions/virtual-environments/blob/7403f3318073a9a0d8ec04eca0cc11e16a3e8196/images/linux/scripts/installers/image-magick.sh#L8-L12
3. https://github.com/actions/virtual-environments/blob/7403f3318073a9a0d8ec04eca0cc11e16a3e8196/images/linux/scripts/installers/subversion.sh#L8

Reply:
then we probably need an extra work item for moving these tools to toolsets

Reply:
Agree with both.
Let's log low-priority item to get rid of simple scripts where we don't need additional logic except "apt install"

Reply:
Done:
https://github.com/actions/virtual-environments-internal/issues/1636